### ISSUE TITLE
Remove init scripts?

### ISSUE THREAD

**Issue by nawatts** [broadinstitute/gnomad_methods#187]
Now that this can be used on a Dataproc cluster using `hailctl dataproc start cluster --packages gnomad`, are `gnomad-init.sh` and `master-init.sh` still necessary?

**Comment by konradjk** [comment]
they won't be, but as this is a pretty big breaking change, lets make sure all pipelines that use this package are fixed before we do remove them

**Comment by lfrancioli** [comment]
yeah, we need to make sure people know about the change and give them some time. That said, it'll feel nice and clean when we remove those scripts  ;)

**Comment by ch-kr** [comment]
need to double check with team to make sure no one is using these before we delete

**Comment by ch-kr** [comment]
closing as these have been removed https://github.com/broadinstitute/gnomad_methods/pull/764
